### DISCUSSION TITLE
Document Q&A Retrieval Chain always answers in english (it should not)

### DISCUSSION THREAD

**Discussion by marius-smg** [FlowiseAI/Flowise#382] [General]
Hey i have a Conversational Retrieval QA set up. the Document in the pinecone database is entirely in german, my additional Paramters are also written in german, in it tell it to explicitly answer in german. however 2 in 3 times it answeres in english. Do any of you know how to solve this issue?

**Comment by HenryHengZJ** [comment]
have a look at this to see if it helps - https://github.com/FlowiseAI/Flowise/issues/384#issuecomment-1601365626

**Reply by marius-smg** [reply]
Thank you, unfortunately i havent found a solution yet. But i want to say Thank you very much for your amazing work and all the help you provide!
